modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -897,6 +941,53 @@ public Comment getScopeScopeKeyByExternalReferenceCodeComment(
 			serviceBuilderComment, _commentManager, PortalUtil.getPortal());
 	}
 
+	@Override
+	public Page<Comment>
+			getScopeScopeKeyByExternalReferenceCodeCommentChildCommentsPage(
+				String scopeKey, String externalReferenceCode,
+				String commentExternalReferenceCode, String search,
+				Aggregation aggregation, Filter filter, Pagination pagination,
+				Sort[] sorts)
+		throws Exception {
+
+		if (!_objectDefinition.isEnableComments() ||
+			!FeatureFlagManagerUtil.isEnabled(
+				_objectDefinition.getCompanyId(), "LPD-69419")) {
+
+			throw new UnsupportedOperationException();
+		}
+
+		ObjectEntry objectEntry = _getObjectEntry(
```

Comment:
Can you create a private method to reuse this duplicate code?

Reply:
Which code is duplicated? It's hard to understand since I moved the methods to another class in the next commits.